translations/stripes-smart-components/ar.json

```diff
@@ -178,7 +178,6 @@
     "reset": "إعادة تعيين",
     "notes.assignUnassignAll": "تعيين / إلغاء تعيين كل الملاحظات",
     "customFields": "حقول مخصصة",
-    "customFields.new": "جديد",
```

Comment:
You only have to make changes to en.json. All other files will be automatically updated by the translation bot after the PR is merged.

Reply:
Fixed in new commit